experimental/Schemes/CoveredProjectiveSchemes.jl

```diff
@@ -281,10 +269,10 @@ function Base.show(io::IO, CPS::CoveredProjectiveScheme)
   n = length(projective_patches(CPS))
   K = base_ring(base_scheme(CPS))
   if get(io, :supercompact, false)
-    print(io, "Covered projective scheme")
+    print(io, "Relative projective scheme")
```

Comment:
Please check this with @HechtiDerLachs -- "Covered projective scheme" has the wording of the underlying type built into it. This might be on purpose.